examples/src/main/kotlin/examples/visualization/LeNetFashionMnistVisualization.kt

```diff
@@ -38,43 +41,59 @@ fun main() {
 
     val (newTrain, validation) = train.split(0.95)
 
-    lenet5().use {
-        it.compile(
+    val sampleIndex = 42
+    val x = test.getX(sampleIndex)
+    val y = test.getY(sampleIndex).toInt()
+
+    lenet5().use { model ->
+
+        model.compile(
```

Comment:
could you please use it instead of model here

Reply:
done